### ISSUE TITLE
The "setuptools" requirement should be explicitly defined (ModuleNotFoundError: No module named 'pkg_resources').

### ISSUE THREAD

**Issue by AlirezaRoshanzamir** [wemake-services/flake8-eradicate#244]
Due to the usage of the "pkg_resources" module, the "setuptools" requirement should be explicitly defined (in this project, it is runtime requirement as well as installation time requirement). Because there are tools (such as [Pants](https://www.pantsbuild.org/)) that, after installing the library, do not maintain the installation time requirements, hence, this error is given:

```
ModuleNotFoundError: No module named 'pkg_resources'
```

which raises this exception on the Flake8:

```
flake8.exceptions.FailedToLoadPlugin: Flake8 failed to load plugin "E8" due to No module named 'pkg_resources'.
```

**Comment by sobolevn** [comment]
@AlirezaRoshanzamir PR is welcome!

**Comment by AlirezaRoshanzamir** [comment]
@sobolevn The [pull-request](https://github.com/wemake-services/flake8-eradicate/pull/245) is created.

**Comment by AlirezaRoshanzamir** [comment]
Thanks for accepting the pull request.

Is there a specific timeline for the next release? Or can this change be released as version 1.2.1?

**Comment by sobolevn** [comment]
1.2.1 is released.
